docs/conf.py

```diff
@@ -54,6 +54,10 @@
 # If your documentation needs a minimal Sphinx version, state it here.
 needs_sphinx = "1.7"
 
+intersphinx_mapping["astropy"] = ("http://docs.astropy.org/en/stable/", None)
```

Comment:
I think intersphinx to astropy is already provided by `from sphinx_astropy.conf.v1 import *` above.

See https://github.com/astropy/sphinx-astropy/blob/6928e75f53f3988e55121da5cd485fadfb2d8455/sphinx_astropy/conf/v1.py#L54 for all available packages.

Reply:
see 8e01ef1